### PR TITLE
UI/15399696 layout components

### PR DESCRIPTION
**Pull request by KatarzynaStefanowiczPagepro**
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

### CODE REVIEW

File: src/components/frame/ThreeColumns/ThreeColumns.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React, { PropsWithChildren } from 'react';
+
+import clsx from 'clsx';
+import styles from './ThreeColumns.module.css';
+import { ThreeColumnsProps } from './types';
+
```
Comment (agnieszkajarosikloj) [inline src/components/frame/ThreeColumns/ThreeColumns.tsx:6]: I think display name is missing here.

File: src/components/frame/TwoColumns/TwoColumns.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React, { PropsWithChildren } from 'react';
+
+import { TwoColumnsProps } from './types';
+
```
Comment (agnieszkajarosikloj) [inline src/components/frame/TwoColumns/TwoColumns.tsx:4]: Same here - display name.

File: src/components/frame/ThreeColumns/ThreeColumns.tsx (hunk 1 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/frame/ThreeColumns/ThreeColumns.tsx]: Added

File: src/components/frame/TwoColumns/TwoColumns.tsx (hunk 2 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/frame/TwoColumns/TwoColumns.tsx]: Added

File: src/stories/frame/TwoColumns.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        <Story />
+      </div>
+    ),
+  ],
+};
+
+export default meta;
+type Story = StoryObj<typeof TwoColumns>;
+
+export const Base: Story = {
```
Comment (krawczykkatarzyna) [inline src/stories/frame/TwoColumns.stories.tsx:23]: please add ThreeColumns to storybook

**Review by agnieszkajarosikloj — APPROVED** [review]

File: src/stories/frame/TwoColumns.stories.tsx (hunk 3 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/stories/frame/TwoColumns.stories.tsx:23]: It's added In ThreeColumns.stories.tsx

File: src/stories/frame/TwoColumns.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import TwoColumns from '~frame/Extensions/TwoColumns';
+
+const meta: Meta<typeof TwoColumns> = {
+  title: 'Frame/Two Columns',
```
Comment (krawczykkatarzyna) [inline src/stories/frame/TwoColumns.stories.tsx:6]: 'Extension /Two Columns'

File: src/stories/frame/ThreeColumns.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+import ThreeColumns from '~frame/Extensions/ThreeColumns';
+
+const meta: Meta<typeof ThreeColumns> = {
+  title: 'Frame/Three Columns',
```
Comment (krawczykkatarzyna) [inline src/stories/frame/ThreeColumns.stories.tsx:6]: 'Extension /Three Columns'

**Review by krawczykkatarzyna — APPROVED** [review]

**Pull request merged**